### ISSUE TITLE
Error using PHP 8.2

### ISSUE THREAD

**Issue by pbking** [sirbrillig/phpcs-changed#64]
Using this library with PHP 8.2 caused the following errors to occur:

```
PHP Deprecated:  Using ${var} in strings is deprecated, use {$var} instead in /Projects/themes/vendor/sirbrillig/phpcs-changed/PhpcsChanged/SvnWorkflow.php on line 21
...
```
(as well as quite a few additional lines)

Changing my version of PHP to 8.0 eliminated those errors.

**Comment by sirbrillig** [comment]
I've [released 2.10.1](https://github.com/sirbrillig/phpcs-changed/releases/tag/v2.10.1). Manual testing with PHP 8.2 does not produce any more errors to appear for me. Could you confirm that, @pbking?

**Comment by sirbrillig** [comment]
I'm going to mark this as resolved unless I get any further information.
